checks/core/heartbeat.py

```diff
@@ -9,15 +9,15 @@
 from poucave.utils import ClientSession, retry_decorator
 
 
-EXPOSED_PARAMETERS = ["url"]
+EXPOSED_PARAMETERS = ["url", "status"]
 
 
 @retry_decorator
-async def run(url: str) -> CheckResult:
+async def run(url: str, status: int = 200) -> CheckResult:
     async with ClientSession() as session:
         try:
             async with session.get(url) as response:
-                status = response.status == 200
+                status = response.status == status
```

Comment:
You are shadowing the integer parameter `status` with a Boolean variable of the same name here, which makes this slightly confusing on first reading. How about renaming the new parameter to `expected_status` instead? This might be a better description of what it does anyway, and make the configuration more readable as well.

Reply:
You're fully right :)